skills/csharp/experimental/eventskill/Responses/ResponsesAndTexts/ResponsesAndTexts.lg

```diff
@@ -0,0 +1,3 @@
+[import](..\FindEvents\FindEventsResponses.lg)
```

Comment:
so what happens when user is in a different locale that doesn't have FindEventsResponses.{that locale}.lg imported, there would be exceptions right? since the code still looks for the same template. should probably do some easy translation with Google translate for those other languages.

Reply:
The skill actually does not work due to the API issue (API not work now). it's also dropped from the experimental skill list in the skill document. So I just updated the skill to align with the SDK change. The response message for FindEvent dialog has the multi-lang support issue but I think we can leave it there until we have further plan to support this skill.